pkg/config/store/queue_test.go

```diff
@@ -139,19 +140,27 @@ func TestQueueCancelSync(t *testing.T) {
 			Value: &BackEndResource{},
 		}
 	}
+	// Wait for the queue's run loop to process chin.
```

Comment:
Is `chin` a thing ? Sorry, I am not clear what it means.

Reply:
Yes it's the input channel created in line 134.